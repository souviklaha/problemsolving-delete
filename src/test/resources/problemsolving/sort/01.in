5
23 38 19 1 22