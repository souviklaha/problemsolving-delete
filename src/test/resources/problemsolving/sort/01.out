1 19 22 23 38